tests/publiccloud/azure_vn.sh

```diff
@@ -0,0 +1,365 @@
+#!/bin/bash -eu
+set -e
```

Comment:
you don't need that when you have `#!/bin/bash -eu`

Reply:
done